62/docs/update readme (#65)
cvpaperchallenge/Ascender: README.md

$ sudo systemctl restart docker
```

> If `sudo docker run --rm --gpus all nvidia/cuda:11.3-base nvidia-smi` works, installation succeeded.

Author: here need to change from `11.3` to `11.0.3`

### Change the Python version to run CI jobs

By default, CI job (GitHub Actions workflow) of Ascender is run against Python 3.8 and 3.9. If you want to change the target Python version, please modify [the matrix part of `.github/workflows/lint-and-test.yaml`](https://github.com/cvpaperchallenge/Ascender/blob/master/.github/workflows/lint-and-test.yaml#L18).

> ### The change to Dockerfile has not been reflected correctly on the image build

Reviewer: ```suggestion
### When changes to the Dockerfile are not reflected
```

> When you run `sudo docker compose up --build` after adding some change on the Dockerfile, you may find no changes has been made on the image built.  This may be because the `docker build` command has used the build cache.  When that happens, try using `sudo docker compose up --build --force-recreate`, instead.

Reviewer: @YoshikiKubotani 
When changes to the Dockerfile are not reflected, potentially there are two possibilities. 
1. Docker using cache at image build step.
2. Docker using already created container.

I think `sudo docker compose up --build --force-recreate`  will solve problem caused by 2, but might not solve caused by 1. So maybe write two command like bellow might be more more friendly: 

- `sudo docker compose build --no-cache`  then `sudo docker compose up` (solution for 1.)
- `sudo docker compose up --force-recreate` (solution for 2.)

Or run `sudo docker compose build --no-cache` then `sudo docker compose up --force-recreate` will solve both case.
